CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The REST API versions in the table are just for your information as the values a
 
 ### Added
 
+- new visualization type - Bullet chart
 - Column auto-sizing in pivot tables can now be configured (applied to visible columns during initial table load).
```

Comment:
Column auto-sizing in pivot tables (applied to visible columns during the initial table load)